packages/katello/katello-repos/katello-repos.spec

```diff
@@ -3,12 +3,12 @@
 %define repo_dir %{_sysconfdir}/yum.repos.d
 %define repo_dist %{dist}
 
-%global prereleasesource nightly
+%global prereleasesource rc1
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
 %global release 3
 
 Name:           katello-repos
-Version:        4.1
+Version:        4.1.0
```

Comment:
Should nightly also be x.y.z?
https://github.com/theforeman/foreman-packaging/blob/d5b11c61ec7da57060f8dc760f4b644e4bd075eb/packages/katello/katello-repos/katello-repos.spec#L11